packages/experimental/mosaic/src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,112 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import faker from 'faker';
+import { Buildings, DotsThree, Envelope, User } from 'phosphor-react';
+import React from 'react';
+
+// TODO(burdon): Add note to @dxos/react-components/plugin
+// storybook config (files and modules)
+// dxosTheme import
+
+import { Button, DensityProvider, getSize } from '@dxos/react-components';
+import '@dxosTheme';
+
+import { Card } from './Card';
+import { CardActions } from './CardActions';
+import { CardContent } from './CardContent';
+import { CardHeader } from './CardHeader';
+
+// TODO(burdon): Copy storybook structure from @dxos/react-components.
+
+// TODO(burdon): IconButton.
+// TODO(burdon): Density: Input, Checkbox.
+// TODO(burdon): ListItem as CardHeader.
+
+// TODO(burdon): Column on mobile; no rounded corner if mobile column.
+
+// TODO(burdon): Factor out default semantic colors.
+
+const styles = {
+  header: 'bg-blue-300 dark:bg-zinc-800'
+};
+
+// TODO(burdon): Test embedded table/lists (e.g., checkbox in side gutter).
+
+const Test = () => {
+  return (
+    <div className='flex flex-col w-full md:w-[390px] m-4 space-y-8'>
+      <DensityProvider density='fine'>
+        <Card slots={{ root: { className: 'text-sm' } }}>
+          <CardHeader
+            slots={{ root: { className: styles.header } }}
+            icon={<User className={getSize(6)} />}
+            action={<DotsThree className={getSize(6)} />}
```

Comment:
Here’s where things get dicey, generally the Radix-style subcomponents are _for_ avoiding props which are literal `React.ReactNode[]`s, which should either be just `props.children` or their own subcomponent so we’re not proliferating standards 3 ways (since `slots.{slot}.children` could also be used for this).

I think these should probably use a similar structure to list item endcaps, where endcaps (or whatever else we should call square layout parts) are used to defer semantics to the developer or other libraries so this package can focus on layout and a coherent API surface.